Vagrantfile

```diff
@@ -1,65 +1,53 @@
 # -*- mode: ruby -*-
-# # vi: set ft=ruby :
+# vi: set ft=ruby :
 
-# This is fork of https://github.com/coreos/coreos-vagrant
+require_relative 'vagrant_ros_guest_plugin.rb'
 
-require 'fileutils'
-
-Vagrant.require_version ">= 1.6.0"
-
-CONFIG = File.join(File.dirname(__FILE__), "vagrant", "config.rb")
-
-# Defaults for config options defined in CONFIG
-$update_channel = "alpha"
+$number_of_nodes = 2
+# DO NOT USE 172.17.*, 172.18.*, 10.0.2 as these are used on other NICs: routing will get screwed!
+$private_ip_prefix = "172.19.8" # rancher-server will be on 172.19.8.100, other nodes will start from 172.19.8.101
 $expose_rancher_ui = 8080
 $vb_gui = false
 $vb_memory = 1024
 $vb_cpus = 1
-$private_ip = "172.17.8.100"
 
-if File.exist?(CONFIG)
-  require CONFIG
-end
-
-Vagrant.configure("2") do |config|
-  config.vm.box = "coreos-%s" % $update_channel
-  config.vm.box_version = ">= 308.0.1"
-  config.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json" % $update_channel
+$rsync_folder_disabled = true
 
-  config.vm.provider :vmware_fusion do |vb, override|
-    override.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant_vmware_fusion.json" % $update_channel
-  end
+Vagrant.configure(2) do |config|
+  config.vm.box         = "rancherio/rancheros"
+  config.vm.box_version = ">=0.3.3"
 
-  config.vm.provider :virtualbox do |v|
-    # On VirtualBox, we don't have guest additions or a functional vboxsf
-    # in CoreOS, so tell Vagrant that so it can be smarter.
-    v.check_guest_additions = false
-    v.functional_vboxsf     = false
-  end
+  (0..$number_of_nodes-1).each do |i|
```

Comment:
You should configure the master before entering the for loop. You'll be able to simplify the logic.